gcp/pubsub/src/main/scala/com/commercetools/queue/gcp/pubsub/PubSubConfig.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Commercetools GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.commercetools.queue.gcp.pubsub
+
+case class PubSubConfig(subscriptionNamePrefix: String)
+
+object PubSubConfig {
+  val default: PubSubConfig = PubSubConfig("fs2-queue")
```

Comment:
Shall we just provide an empty string as a default?
I'm not sure what was the reasoning behind this choice, are there any implications on leaving this empty? @satabin maybe know more.

```suggestion
  val default: PubSubConfig = PubSubConfig("")
```

Reply:
TBH as an end user I would still prefer to have some kind of prefix instead of not having one, but I'm happy to discuss the best option with you and @satabin

Reply:
I would tend to agree that as a user I'd like a non empty prefix, to avoid collision with potential other queues. In any case it needs to be documented on the GCP specific page.